src/registrar/tests/test_models.py

```diff
@@ -654,3 +654,48 @@ def test_identity_verification_with_new_user(self):
         """A new user who's neither transitioned nor invited should
         return True when tested with class method needs_identity_verification"""
         self.assertTrue(User.needs_identity_verification(self.user.email, self.user.username))
+
+
+class TestContact(TestCase):
+    def setUp(self):
+        self.email = "mayor@igorville.gov"
+        self.user, _ = User.objects.get_or_create(email=self.email, first_name="Rachid", last_name="Mrad")
```

Comment:
**Comment (non-blocking)**: It's a little unusual for us to use our actual first and last names in our test cases. Should this just be "John Doe" or "Firstname Lastname" instead?

Reply:
Jeff Lebowski it is